torch/csrc/api/src/nn/modules/rnn.cpp

```diff
@@ -996,14 +1006,29 @@ GRUCellImpl::GRUCellImpl(const GRUCellOptions& options_)
       options(options_) {}
 
 Tensor GRUCellImpl::forward(const Tensor& input, Tensor hx) {
-  this->check_forward_input(input);
+  this->check_forward_input(input, "input");
+  this->check_forward_input(hx, "hidden");
```

Comment:
same comment as above here about `hx.defined()`

Reply:
the same as above.